diff --git a/node_modules/svelte-lexical/dist/components/toolbar/MoreStylesDropDown/ClearFormattingDropDownItem.svelte b/node_modules/svelte-lexical/dist/components/toolbar/MoreStylesDropDown/ClearFormattingDropDownItem.svelte
index c98cfa5..c8dc77d 100644
--- a/node_modules/svelte-lexical/dist/components/toolbar/MoreStylesDropDown/ClearFormattingDropDownItem.svelte
+++ b/node_modules/svelte-lexical/dist/components/toolbar/MoreStylesDropDown/ClearFormattingDropDownItem.svelte
@@ -1,6 +1,6 @@
 <script lang="ts">import DropDownItem from "../../generic/dropdown/DropDownItem.svelte";
 import { getActiveEditor } from "../../../core/composerContext.js";
-import { $isDecoratorBlockNode as isDecoratorBlockNode } from "$lib/core/plugins/DecoratorBlockNode.js";
+import { $isDecoratorBlockNode as isDecoratorBlockNode } from "../../../core/plugins/DecoratorBlockNode.js";
 import {
   $isQuoteNode as isQuoteNode,
   $isHeadingNode as isHeadingNode
